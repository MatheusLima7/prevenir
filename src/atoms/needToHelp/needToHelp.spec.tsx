import React from "react";
import { render } from "@testing-library/react-native";

import NeedToHelp from "./needToHelp";

describe("NeedToHelp Component", () => {
  it("should render component correctly", () => {
    const container = render(
      <NeedToHelp title={"need_to_help"} subtitle={"press_here"} />
    );

    expect(container).toMatchSnapshot();
  });
});
