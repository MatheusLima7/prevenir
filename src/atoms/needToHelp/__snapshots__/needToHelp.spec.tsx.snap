// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NeedToHelp Component should render component correctly 1`] = `
<View
  style={
    {
      "display": "flex",
      "flexDirection": "row",
      "justifyContent": "center",
      "marginTop": 60,
      "textAlign": "center",
    }
  }
>
  <Text
    size="small"
    style={
      [
        {
          "color": undefined,
          "display": "flex",
          "fontFamily": "Quicksand-Regular",
          "fontSize": 14,
          "fontWeight": "400",
          "lineHeight": 20,
        },
        {},
      ]
    }
    testId="text-component"
    type="text"
  >
    need_to_help
     
  </Text>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "opacity": 1,
      }
    }
  >
    <Text
      color="blue"
      size="small"
      style={
        [
          {
            "color": "#0081A7",
            "display": "flex",
            "fontFamily": "Quicksand-SemiBold",
            "fontSize": 14,
            "fontWeight": "600",
            "lineHeight": 20,
          },
          {},
        ]
      }
      testId="text-component"
      type="title"
    >
      press_here
    </Text>
  </View>
</View>
`;
